Solution1/Solution1.Tests/Integration/WeatherApiTests.cs

```diff
@@ -18,11 +18,12 @@ namespace Weather.Tests.Integration
     public class WeatherApiTests
     {
         private readonly JsonSerializerOptions _serializerOptions;
-        private readonly string cityName = "Minsk";
-        private readonly int countDays = 3;
-        private readonly string currentWeatherURL = "/api/weather/current?";
-        private readonly string forecastWeatherURL = "/api/weather/forecast?";
-        private readonly List<string> comments = new List<string>() { "Dress warmly.", "It's fresh.", "Good weather.", "It's time to go to the beach." };
+        private readonly string _cityName = "Minsk";
+        private readonly int _countDays = 3;
+        private readonly string _currentWeatherURL = "/api/weather/current?cityName=";
```

Comment:
put urls for weather api from Constants to configuration, both for console and web apps

Reply:
do you need these a fields?